flang/lib/Lower/Bridge.cpp

```diff
@@ -1106,9 +1106,10 @@ class FirConverter : public Fortran::lower::AbstractConverter {
   }
   void genFIR(const Fortran::parser::InquireStmt &stmt) {
     auto iostat = genInquireStatement(*this, stmt);
-    genIoConditionBranches(
-        getEval(), std::get<std::list<Fortran::parser::InquireSpec>>(stmt.u),
-        iostat);
+    if (const auto *specs =
+            std::get_if<std::list<Fortran::parser::InquireSpec>>(&stmt.u)) {
+      genIoConditionBranches(getEval(), *specs, iostat);
+    }
```

Comment:
Braces not needed per LLVM coding guidelines. https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
So fir-dev follows `LLVM coding standard` from next time I shall check with above link.

Reply:
Pedantically speaking, we're using MLIR coding conventions here. Lowering and the fir passes are building and built upon, resp., MLIR and are thus fundamentally intertwined with those sources.